source/connection-strings/mongodb-database-connection-strings.txt

```diff
@@ -1,3 +1,125 @@
+.. _rm-mongodb-database-connection-strings:
+
 ===================================
 MongoDB Database Connection Strings
 ===================================
+
+.. include:: /includes/uri-usage-disclaimer.rst
+
+To start a migration job, Relational Migrator must connect to your MongoDB 
+database. Relational Migrator requires a username and password which has ``readWrite`` 
+access in the target database. This page describes 
+the procedure to make a authenticated user account and the Uniform Resource 
```

Comment:
(typo)
s/b "... to make [an] authenticated user ..."

Reply:
Updated.